caffe2/utils/math/reduce.cc

```diff
@@ -14,12 +8,19 @@
 #include <mkl.h>
 #endif // CAFFE2_USE_MKL
 
+#include <c10/util/accumulate.h>
 #include "caffe2/core/context.h"
 #include "caffe2/utils/eigen_utils.h"
 #include "caffe2/utils/math.h"
 #include "caffe2/utils/math/elementwise.h"
 #include "caffe2/utils/math/utils.h"
 
+#include <algorithm>
```

Comment:
Why are all these new imports added if you only add usage for `c10::multiply_integers` ?